index.js

```diff
@@ -133,10 +140,34 @@ if (context.BOT_FUNCTION) {
 context.BOT_FUNCTION = function ticker(delta) {
     delta /= 100;
 
+    if(GAME.m_IsStateLoading) {
+        return;
+    }
+
+    if(GAME.m_State.m_bRunning === false) {
```

Comment:
if this switches back to the zone select screen you shouldn't need this anymore

Reply:
Yes lines 147-149 is switching back to the zone screen

Reply:
Alright, I will pull this and fix it locally